test/unit/specs/components/DecisioningEngine/createOnResponseHandler.spec.js

```diff
@@ -230,7 +266,7 @@ describe("DecisioningEngine:createOnResponseHandler", () => {
       })
     };
 
-    const decisionContext = {};
+    const decisionContext = DECISION_CONTEXT;
```

Comment:
This test does not actually test the code you wrote in `createDecisioningContext`.  we need to test context generated by it instead of mocking the whole context like this.